sdk/identity/identity/test/public/chainedTokenCredential.spec.ts

```diff
@@ -48,6 +48,12 @@ describe("ChainedTokenCredential", function() {
     const error = await getError<AggregateAuthenticationError>(
       chainedTokenCredential.getToken("scope")
     );
-    assert.equal(error.errors.length, 2);
+    assert.deepEqual(error.errors.length, 2);
+    assert.deepEqual(
+      error.message,
+      `ChainedTokenCredential authentication failed.
+CredentialUnavailableError: unavailable.
+CredentialUnavailableError: unavailable.`
```

Comment:
Turns out this error structure was slightly bad. We didn’t have a test for the full error message. Now we do!